pages/vi/faq.md

```diff
@@ -103,7 +103,7 @@
 * Reference/Advanced
     - [Git Pro (Book)](https://git-scm.com/book/en/v2)
     - [GitHub Tips](https://github.com/git-tips/tips/blob/master/README.md)
-    - [Oh, Shit, Git](http://ohshitgit.com/)
+    - [The Website called...](http://ohshitgit.com/)
```

Comment:
what we want is
`    - The Website called...[Oh, S***, Git](http://ohshitgit.com/)`